### ISSUE TITLE
Unviewed keys are not being indicated in the tab icon.

### ISSUE THREAD

**Issue by revilheart** [revilheart/ESGST#258]
[Source](https://www.steamgifts.com/go/comment/baqZBO8)

**Comment by ldeman** [comment]
hmm... wish i woulda read this earlier... pretty sure i didn't get any color personally..  it still was flashing though, but i think that's cg's implementation if i'm not mistaking.

**Comment by revilheart** [comment]
Yeah, it wasn't working, but it is now, I tested with my last win. It doesn't get colored anymore though, a trophy icon appears in the title now.
